src/Ledger/Foreign/HSLedger.agda

```diff
@@ -223,7 +230,7 @@ instance
         ; maxTxSize        = maxTxSize
         ; maxHeaderSize    = maxHeaderSize
         ; maxValSize       = maxValSize
-        ; minUTxOValue     = minUTxOValue
+        ; minUtxOValue     = minUtxOValue
```

Comment:
I think, for consistency, we are changing all appearances of "UTxO" in a variable name to `Utxo` (but I may be wrong about this).

Reply:
This is another instance where cleanup was undone: I renamed it to `UTxO` recently. I'm in principle OK with naming it `Utxo`, but note that this changes it to `UtxO` which is definitely bad.